docs/manual.rst

```diff
@@ -5,12 +5,32 @@
 Manual
 ******
 
+Learn what is Open Federated Learning (|productName|):
```

Comment:
Awkward phrasing. Change to `What is Open Federated Learning (|productName|)`

Reply:
Fix phrasing as suggested.